### ISSUE TITLE
End line and end column

### ISSUE THREAD

**Issue by glen-84** [sasstools/sass-lint#665]
It would be nice if the rules could specify both start *and* end values (a range) for the line/column, so that linter extensions can properly indicate the problem.

**Comment by bgriffith** [comment]
Yeah that makes sense, I'll label it as an enhancement. Thanks for your contribution.
